### DISCUSSION TITLE
Update marker position

### DISCUSSION THREAD

**Discussion by Sisco21** [fleaflet/flutter_map#951] [Q&A]
Hello,

I'm wondering how i can update a marker position in live?

Best regards,

François

**Comment by ibrierley** [comment]
Probably need a bit more info, but there's an example in the examples folder, live_location (and others) https://github.com/fleaflet/flutter_map/tree/master/example/lib/pages which may do what you want.

**Comment by ibrierley** [comment]
Moving Markers is another example in there. Just depends if you want to move the map as well really.

**Reply by Sisco21** [reply]
Thx it's working now. i found how to do it.

Best regards,

**Comment by fvisticot** [comment]
The provided sample shows how to move / update markers position but a rebuild of the map is done each time a marker is updated.
Is there a better solution with a markerController for better performance rendering ?

I have tested this solution with multiple updates per second and there is a lot of blink

**Comment by ibrierley** [comment]
Rebuilds of the map are normally pretty fast. Sure there isn't something else amiss ? I'm not aware of any way to adjust the position of a marker without a rebuild (maybe there is a way to cause the whole marker layer to get rebuilt via its stream? I've never tried that though to know if its possible).

**Comment by GianfryPeres** [comment]
Sorry if I bother you.
I have to thank for your library.
Mainly because I'm making apps for the disabled and apps for trail enthusiasts.
I also have a site in progress: www.abbici.org/dovesto/
From an app or from the browser I can notify my current location.
I needed a lot!
Thanks again!

@}-,-'-----
Gianfranco
